README.md

```diff
@@ -53,6 +53,48 @@ There is a [live editor](https://data-driven-forms.org/live-editor) where you ca
 The `POST` endpoint will receive the submitted values
 
 ## Dev Instructions
-
 * `npm install`
-* Run both `npm run start` & `SPANDX_CONFIG=path/to/settings-frontend/profiles/local-frontend.js sh path/to/insights-proxy/scripts/run.sh`
+
+### Proxy/Routing
+#### Using insights [frontend-components](https://github.com/RedHatInsights/frontend-components/tree/master/packages/config)
+* Edit `dev.webpack.config.js` to add route for local backend API server(If needed)
+  
+Example: 
+``` javascript
+  const { config: webpackConfig, plugins } = config({
+    rootFolder: resolve(__dirname, '../'),
+    debug: true,
+    useFileHash: false,
+    deployment: process.env.BETA ? 'beta/apps' : 'apps',
+    useProxy: true,
+    appUrl: `/beta/settings/applications`,
```

Comment:
`appUrl: process.env.BETA ? '/beta/settings/applications' : '/settings/applications'`
  
  Can you please change it to this?

Reply:
Done

Reply:
Would you like to see this in both examples?